CIScripts/TestIntegration.ps1

```diff
@@ -0,0 +1,30 @@
+. $PSScriptRoot\InitializeCIScript.ps1
```

Comment:
Dot sourcing allows to run it in the same scope (instead of creating new, child scope for all variables in case of running it with `&`)